vnext/ReactWindowsCore/ReactWindowsCore.vcxproj

```diff
@@ -98,6 +98,14 @@
       </AdditionalIncludeDirectories>
       <AdditionalIncludeDirectories Condition="'$(USE_HERMES)'=='true'">$(HERMES_Package)\installed\$(VcpkgTriplet)\include\;%(AdditionalIncludeDirectories)</AdditionalIncludeDirectories>
       <PreprocessToFile>false</PreprocessToFile>
+      <AdditionalOptions Condition="'$(Configuration)|$(Platform)'=='Debug|ARM'">/await /permissive- %(AdditionalOptions)</AdditionalOptions>
```

Comment:
If adding `/await` here is a problem, it's pretty easy to write this code using the threadpool APIs directly. But coroutines are cool so I wanted to use'em. 🙂